contrib/debian/control

```diff
@@ -36,18 +37,7 @@ Architecture: any
 Depends: adduser,
          libcap2-bin (>= 1:2.0),
          lsb-base (>= 3.1-23.2),
-         zlib1g,
-         libuuid1,
-         libuv1,
-         liblz4-1,
-         libjudydebian1,
          openssl,
-         libmnl0,
-         libjson-c3,
-         libnetfilter-acct1,
-         libprotobuf-c1,
-         libsnappy1v5,
-         libprotoc10,
```

Comment:
Why are we removing these dependencies? I _assume_ they are not required, but some of these look like ones we need for Netdata to run (_though maybe different names?_)

Reply:
Yes, you definitely need them to run, but they are added automatically by them build system since they are used to link against and `${shlibs:Depends}` is used. See https://www.debian.org/doc/manuals/maint-guide/dreq.en.html about this.

You only have to fill in explicit depends which cannot be found automatically. For instance, if netdata would be useless without having a webserver running, you would add an dependency on `httpd` here.

Reply:
Ahh I see! Debian packaging magic™ 🤔 Very good! I come from a [CRUX](https://crux.nu) background :)